javascript-1/script3.js

```diff
@@ -0,0 +1,22 @@
+console.log("EXERCITIUL 3");
+
```

Comment:
Din nou, aici tu pui valorile din variabile, in loc sa pui variabile, daca te uiti la exercitiul 1 si 2 o sa vezi ca ai folosit variabilele a si b, la fel si aici trebuie sa foloseti variabilele luni,marti, miercuri... si sa le adaugi pe ele in text. Te rog sa modifici